modules/041-linstor/templates/linstor-node/linstorsatelliteset.yaml

```diff
@@ -62,9 +62,9 @@ spec:
   resources:
     requests:
       {{- include "helm_lib_module_ephemeral_storage_only_logs" . | nindent 6 }}
-{{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+      {{- if not ( .Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
```

Comment:
to discuss on retro.

Reply:
internally we did not come to a consensus about if/else constructions, this module is constructed with this style.
I suggest to keep it as is until we'll have common decision and refactor this module afterwards.